### PR TITLE
Replace Classic DNS with CIS

### LINKED ISSUE #496: IBM Cloud Classic DNS service is deprecated use CIS instead
Deprecation notice can be referred at https://cloud.ibm.com/docs/dns?topic=dns-register-a-new-domain
Need to use `Cloud Internet Services (CIS)` instead.

### PR DESCRIPTION
**Pull request by yussufsh**
Fixes #496

### CODE REVIEW

**Comment by yussufsh** [comment]
@Prajyot-Parab please help with the test and review. Thanks!

File: variables.tf [hunk 1]
```diff
@@ -219,7 +219,7 @@ variable "rhel_smt" {
 ################################################################
 variable "use_ibm_cloud_services" {
   type        = bool
-  description = "Experimental: Flag to use IBM Cloud DNS and VPC Loadbalancer instead of bastion services. Please set variables setup_snat=true and setup_squid_proxy=false"
+  description = "Experimental: Flag to use Internet Services (CIS) and Loadbalancer services on VPC instead of bastion services. Please set variables setup_snat=true and setup_squid_proxy=false"
```
[the remark this answers is not in the document]
Comment (yussufsh) [reply variables.tf:222]: Nope.... If ibmcloud services are enabled we use VPC LoadBalancer and haproxy/named on bastion is masked off.

**Comment by yussufsh** [comment]
@Prajyot-Parab any update?

**Comment by Prajyot-Parab** [comment]
> @Prajyot-Parab any update?

I tried multiple times few days back, was not able to get a fully successful run, will try again

**Comment by yussufsh** [comment]
> > @Prajyot-Parab any update?
> 
> I tried multiple times few days back, was not able to get a fully successful run, will try again

Please add the details at https://github.com/ocp-power-automation/ocp4-upi-powervs/issues/496. I will take a look.
Thanks.

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Pull request merged**